### PR TITLE
Find mixins assigned to namespaces

### PR DESCRIPTION
**Pull request by justinfagnani**
- [ ] CHANGELOG.md has been updated

### CODE REVIEW

File: src/generate-elements.ts [hunk 1]
```diff
@@ -64,52 +80,66 @@ export function validateElements(analyzedPackage: Elements|null|undefined) {
...
+    mixin: ResolvedMixin, packagePath: string): ElementMixin {
+  const metadata: ElementMixin =
+      serializeElementLike(mixin, packagePath) as ElementMixin;
+  metadata.name = mixin.name;
+  return metadata;
+}
+
+function serializeElementLike(
+    elementOrMixin: ElementOrMixin, packagePath: string): ElementLike {
+  // if (!resolvedElement.tagName) {
```
Comment (rictic) [inline src/generate-elements.ts]: Move this up to serializeElement?

File: src/model/element-base.ts [hunk 2]
```diff
@@ -58,8 +70,26 @@ export abstract class ElementBase implements Feature {
   astNode: estree.Node|null;
   kinds: Set<string> = new Set(['element']);
   warnings: Warning[] = [];
+  slots:
```
Comment (rictic) [inline src/model/element-base.ts]: Recommend:

```typescript
'slots': Slot[] = [];
```

File: src/generate-elements.ts (hunk 1 above)
Comment (justinfagnani) [inline src/generate-elements.ts]: Removing. We need to document abstract elements with no tagName

File: src/model/element-base.ts (hunk 2 above)
Comment (justinfagnani) [inline src/model/element-base.ts]: done

**Pull request merged**